lesson16.md

```diff
@@ -0,0 +1,114 @@
+<!-- .slide: id="lesson16" -->
+
+# Basic Frontend - Fall 2023
+
+Lesson 16, Tuesday, 2023-11-21
+
+---
+
+Today's lesson will be the last lesson before starting with the project, and we will be doing a small project to get you
+up and running!
+
+The structure is very loose, it is up to you to explore and find what you enjoy!
+
+---
+
+## Dogs
+
+https://dog.ceo/dog-api/documentation/random
+> Display a random dog image
+
+https://kinduff.github.io/dog-api/
+> Fetch dog facts
+
+---
+
+### Task
+
+Create a simple webpage, that fetches dog images from the API. Pair it with the second API and show some dog facts
+together with images.
+
+---
+
+## Holidays
+
+https://date.nager.at/Api
+> Query the holidays of over 100 countries
+
+---
+
+### Task
+
+Show the holidays in Germany for 2024.
+This differs from state to state. Add a dropdown to select a specific state?
```

Comment:
I wouldn't phrase it as a question. Maybe as a bonus task, like:

BONUS: Add a dropdown to select holidays for one state only.

Reply:
Said and done!